### ISSUE TITLE
Remove legacy git tags v1.4.0-pre-alpha-1 and v1.4.0-pre-alpha-2

### ISSUE THREAD

**Issue by navidshaikh** [vmware-tanzu/tanzu-framework#1564]
**Bug description**
git tags `v1.4.0-pre-alpha-1` and `v1.4.0-pre-alpha-2` were created before we started tanzu-framework v0.1.0 release. There are no releases cut from the mentioned tags. These tags are essentially the higher versions w.r.t to semver from the latest release of tanzu-framework (v0.15.0) at the time of filing this issue. We've https://github.com/vmware-tanzu/tanzu-framework/issues/1556 filed to track retracting these tags when working with go modules, however at later point, these tags will conflict with tanzu-framework releases and should be removed.

**Affected product area (please put an X in all that apply)**

- [ ] APIs
- [ ] Addons
- [ ] CLI
- [ ] Docs
- [ ] IAM
- [ ] Installation
- [ ] Plugin
- [ ] Security
- [x] Test and Release
- [ ] User Experience

**Expected behavior**
tanzu-framework tag versions are following semver guidelines.

**Steps to reproduce the bug**

**Version** (include the SHA if the version is not obvious)

**Environment where the bug was observed (cloud, OS, etc)**

**Relevant Debug Output (Logs, manifests, etc)**

**Comment by navidshaikh** [comment]
@vmware-tanzu/tanzu-framework-release-team

**Comment by navidshaikh** [comment]
The `retract` line should be removed from go.mod when the git tags are removed - see https://github.com/vmware-tanzu/tanzu-framework/pull/1557

**Comment by navidshaikh** [comment]
@ZihanJiang96 : Are the mentioned tags referenced in the downstream? Either way, we'll need to take these tags off from the upstream repo as it'd come in the way of the releases as we progress.

**Comment by navidshaikh** [comment]
The mentioned tags are being referred and can not be removed.
